app/models/user.rb

```diff
@@ -52,6 +52,8 @@ def authenticatable_salt
     "#{super}#{session_token}"
   end
 
+  # Further work required on this as soon as the frontend is decommissioned.
```

Comment:
I would actually mark this as a `TODO: `. Tools like Atom, VSCode etc have features that allow you to quickly find any TODO's in a project.

This might be overlooked as just a comment unless someone knows to go back to work on this specific bit.